vector/src/main/res/values-fr/strings.xml

```diff
@@ -788,79 +460,39 @@
     <string name="groups_header">Communautés</string>
-    <string name="no_group_placeholder">Aucun groupe</string>
-    <string name="start_new_chat_prompt_msg">Voulez-vous vraiment créer une nouvelle discussion avec %s \?</string>
-    <string name="start_voice_call_prompt_msg">Voulez-vous vraiment engager un nouvel appel audio \?</string>
-    <string name="start_video_call_prompt_msg">Voulez-vous vraiment engager un nouvel appel vidéo \?</string>
-    <string name="groups_list">Liste des Groupes</string>
-    <string name="room_participants_ban_prompt_msg">Exclure un utilisateur va l’expulser de la conversation et l’empêcher de le rejoindre à nouveau.</string>
-    <string name="action_exit">Quitter</string>
-    <string name="room_settings_all_messages_noisy">Notification sonore pour chaque message</string>
+    <string name="start_voice_call_prompt_msg">Voulez-vous vraiment engager un nouvel appel audio \?</string>
+    <string name="start_video_call_prompt_msg">Voulez-vous vraiment engager un nouvel appel vidéo \?</string>
+    <string name="room_participants_ban_prompt_msg">Bannir un utilisateur va l’expulser du salon et l’empêcher de le rejoindre à nouveau.</string>
     <string name="room_settings_all_messages">Notification pour chaque message</string>
-    <string name="room_settings_mention_only">Seulement les mentions</string>
-    <string name="room_settings_mute">Pas de notification</string>
     <string name="room_settings_add_homescreen_shortcut">Ajouter à l’écran d’accueil</string>
     <string name="settings_inline_url_preview">Aperçu des liens</string>
     <string name="settings_vibrate_on_mention">Vibrer lorsque l’on mentionne un utilisateur</string>
-    <string name="room_settings_room_notifications_title">Notifications</string>
-    <string name="room_settings_no_flair">Ce salon n’a de badge pour aucune communauté</string>
-    <string name="room_settings_add_new_group">Nouvel identifiant de communauté (ex. +foo:matrix.org)</string>
-    <string name="room_settings_invalid_group_format_dialog_title">Identifiant de communauté invalide</string>
-    <string name="room_settings_invalid_group_format_dialog_body">« %s » n’est pas un identifiant de communauté valide</string>
     <string name="create">Créer</string>
-    <string name="create_community">Créer une communauté</string>
-    <string name="community_name">Nom de la communauté</string>
-    <string name="community_name_hint">Exemple</string>
-    <string name="community_id">Identifiant de communauté</string>
-    <string name="community_id_hint">Exemple</string>
     <string name="group_details_home">Accueil</string>
-    <string name="group_details_people">Personnes</string>
-    <string name="group_details_rooms">Salons</string>
-    <string name="no_users_placeholder">Pas d’utilisateur</string>
-    <string name="rooms">Conversations</string>
-    <string name="joined">Rejoint</string>
+    <string name="rooms">Conversation</string> <!-- Tchap: Custom string -->
```

Comment:
@Claire1817 why there is no 's' in this translation
I would expect "Conversations"